backend/logic/users/users.js

```diff
@@ -167,10 +169,76 @@ var getCompletedCourses = async studentID => {
   return courses;
 };
 
+var login = async () => {
```

Comment:
nit: inconsistent tabbing (use either 2 or 4 spaces but not both in the same file)

Reply:
@GarethDPeters  You are using Prettier right? you can try turn on its format-onsave feature

Reply:
no reason, but all of our other functions are vars rather than lets. Was the only reason i did it

Reply:
@GarethDPeters I'd say for future reference, use let please :)